### PR TITLE
feat: add bulletin card samples

### PR DESCRIPTION
**Pull request by tingchun0113**
## Types of changes

* **New feature**

## Description

Contribute some bulletin card samples first to the project

### CODE REVIEW

File: components/bulletins/BulletinCard.vue [hunk 1]
```diff
@@ -43,69 +47,76 @@ export default {
...
 
+.bulletinCard_icon {
+    @apply absolute transform scale-90 md:scale-100;
+    top: 12.5%;
+}
+
 .bulletinCard__content {
     @apply absolute;
+    top: 30%;
+    padding: inherit;
```
Comment (josix) [inline components/bulletins/BulletinCard.vue:98]:
```suggestion
```

File: components/bulletins/BulletinCard.vue [hunk 2]
```diff
@@ -43,69 +47,76 @@ export default {
...
 }
 
 .bulletinCard__content > h2 {
     @apply text-base text-center mt-6 mb-5 font-bold;
-    @media (min-width: 415px) {
+    font-family: 'Noto Serif TC';
```
Comment (josix) [inline components/bulletins/BulletinCard.vue:103]:
```suggestion
    @apply font-serif;
```

**Review by josix — CHANGES_REQUESTED** [review]

File: components/bulletins/BulletinCardCollection.vue [hunk 3]
```diff
@@ -36,15 +37,6 @@ export default {
...
-    @apply md:m-4;
-}
-
-.bulletinCardCollection__cardContainer > .bulletinCard {
-    margin: 3px;
-    @media (min-width: 415px) {
-        margin: 30px 20px;
-        padding: 5px;
-    }
+    @apply grid place-items-center gap-10 grid-cols-1 mt-9 md:grid-cols-3 md:pl-14 md:mt-0 md:w-10/12;
```
[the remark this answers is not in the document]
Comment (tingchun0113) [reply components/bulletins/BulletinCardCollection.vue:40]: Fixed

File: components/bulletins/BulletinCard.vue (hunk 1 above)
Comment (tingchun0113) [reply components/bulletins/BulletinCard.vue:98]: Fixed

File: components/bulletins/BulletinCard.vue (hunk 2 above)
Comment (tingchun0113) [reply components/bulletins/BulletinCard.vue:103]: Fixed

File: components/bulletins/BulletinCard.vue [hunk 4]
```diff
@@ -1,43 +1,21 @@
 <template>
-    <div v-show="!!title" :class="classObject">
-        <div class="dummy"></div>
+    <div v-show="!!title" :class="classObject" @click="onClick">
+        <div class="bulletin--card__dummy"></div>
```
Comment (mattwang44) [inline components/bulletins/BulletinCard.vue:3]: most class names do not follow BEM naming convention.

**Comment by mattwang44** [comment]
I prefer merging this PR first. Let's deal with BEM in another PR

**Pull request merged**